src/PVRIptvData.h

```diff
@@ -35,10 +35,17 @@ struct PVRIptvEpgEntry
   int         iChannelId;
   int         iGenreType;
   int         iGenreSubType;
+  int         iYear;
```

Comment:
You seem not to fill the `iYear` member, so no need to add this member.

Reply:
Do you mean you don’t understand how to populate/parse iYear?